dashboard/app/routes/api/curated_inferences/count.route.ts

```diff
@@ -0,0 +1,49 @@
+import type { LoaderFunctionArgs } from "react-router";
```

Comment:
This doesn't need to be a separate, explicitly-defined route. Just abstract it away in a `loader` or `action` in the primary route.

Reply:
https://www.jacobparis.com/content/remix-multiple-actions

Reply:
@GabrielBianconi per our conversation I'm gonna leave this as is -- we will need to be organized about helper API routes but I think this is preferred to reimplementing a router in the action of route modules which need additional data without a form submission.

Reply:
sgtm. let's ask an expert.